[2023-03-21 12:30:17,215] {taskinstance.py:877} INFO - Dependencies all met for <TaskInstance: First_dag_v3.second_task 2023-03-21T12:30:13.321292+00:00 [queued]>
[2023-03-21 12:30:17,231] {taskinstance.py:877} INFO - Dependencies all met for <TaskInstance: First_dag_v3.second_task 2023-03-21T12:30:13.321292+00:00 [queued]>
[2023-03-21 12:30:17,233] {taskinstance.py:1068} INFO - 
--------------------------------------------------------------------------------
[2023-03-21 12:30:17,235] {taskinstance.py:1069} INFO - Starting attempt 1 of 6
[2023-03-21 12:30:17,237] {taskinstance.py:1070} INFO - 
--------------------------------------------------------------------------------
[2023-03-21 12:30:17,270] {taskinstance.py:1089} INFO - Executing <Task(BashOperator): second_task> on 2023-03-21T12:30:13.321292+00:00
[2023-03-21 12:30:17,280] {standard_task_runner.py:52} INFO - Started process 69 to run task
[2023-03-21 12:30:17,292] {standard_task_runner.py:76} INFO - Running: ['airflow', 'tasks', 'run', 'First_dag_v3', 'second_task', '2023-03-21T12:30:13.321292+00:00', '--job-id', '9', '--pool', 'default_pool', '--raw', '--subdir', 'DAGS_FOLDER/first_dag.py', '--cfg-path', '/tmp/tmpg4_z4z1d', '--error-file', '/tmp/tmpddu1ghpt']
[2023-03-21 12:30:17,294] {standard_task_runner.py:77} INFO - Job 9: Subtask second_task
[2023-03-21 12:30:17,470] {logging_mixin.py:104} INFO - Running <TaskInstance: First_dag_v3.second_task 2023-03-21T12:30:13.321292+00:00 [running]> on host 8c52af15d172
[2023-03-21 12:30:17,718] {taskinstance.py:1283} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=Mario
AIRFLOW_CTX_DAG_ID=First_dag_v3
AIRFLOW_CTX_TASK_ID=second_task
AIRFLOW_CTX_EXECUTION_DATE=2023-03-21T12:30:13.321292+00:00
AIRFLOW_CTX_DAG_RUN_ID=manual__2023-03-21T12:30:13.321292+00:00
[2023-03-21 12:30:17,727] {bash.py:135} INFO - Tmp dir root location: 
 /tmp
[2023-03-21 12:30:17,734] {bash.py:158} INFO - Running command: echo Im the second task and the first task executed first
[2023-03-21 12:30:17,803] {bash.py:169} INFO - Output:
[2023-03-21 12:30:17,881] {bash.py:173} INFO - Im the second task and the first task executed first
[2023-03-21 12:30:17,886] {bash.py:177} INFO - Command exited with return code 0
[2023-03-21 12:30:18,040] {taskinstance.py:1192} INFO - Marking task as SUCCESS. dag_id=First_dag_v3, task_id=second_task, execution_date=20230321T123013, start_date=20230321T123017, end_date=20230321T123018
[2023-03-21 12:30:18,174] {taskinstance.py:1246} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2023-03-21 12:30:18,226] {local_task_job.py:146} INFO - Task exited with return code 0
